src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterType } from '../../types/FilterType';
+
+type TodoFilterProps = {
+  todos: Todo[];
+  filterType: FilterType;
+  setFilterType: (filterType: FilterType) => void;
+  activeTodosCount: number;
+  onClearCompleted: () => void;
+};
+
+export const TodoFilter: React.FC<TodoFilterProps> = (
+  {
+    todos,
+    filterType,
+    setFilterType,
+    activeTodosCount,
+    onClearCompleted,
+  },
+) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${activeTodosCount} items left`}
```

Comment:
Consider using a template literal for better readability: `{activeTodosCount} items left` 📖